### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by pataquets**
Add Dockerfile to enable image building.
Using the official Python image, latest 2.x series tag. Perhaps minimalist 'alpine' image could be used, but not sure of the tradeoffs. More info at https://hub.docker.com/_/python/

Just adding files and setting working dir. Default option if none provided is `--help` to show usage. Hostnames such as `localhost` won't work, since nothing else apart from `redis-statsd.py` runs inside the container, you have to look for the Redis server elsewhere outside the container.

Build:

```
$ docker build -t redis-statsd .
```

Run (assuming an already running Redis):

```
$ docker run -dit --name redis-statsd redis-statsd [options...]
```

FYI, there's a still quicker to test, already built image on my Docker Hub. Test it by running:

```
$ docker run --rm -it --name redis-statsd pataquets/redis-statsd [options...]
```

Using `--rm` instead of `-d` makes the container not go background. Stop it by `CTRL+C`'ing it.

Optional improvement to come:
- Create an 'official', based on your repo, automated build at Docker Hub for the image: https://docs.docker.com/docker-hub/builds/ . Just requires a free Docker Hub account and a following a quick 'Create automated build' process.

### CODE REVIEW

**Comment by pataquets** [comment]
Bump.

**Comment by pataquets** [comment]
@gphat : Ping. Are you still the maintainer?

File: Dockerfile
Comment (gorzell) [inline Dockerfile]: `COPY` is preferred over add, ref the [best practices guide](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy).  Also this, copies the README.md into the location as well, which is necessary, so just be explicit.

File: Dockerfile
Comment (gorzell) [inline Dockerfile]: I am curious about the directory structure.  Why not `/redis-statsd/`?

**Review by gorzell — CHANGES_REQUESTED** [review]
Couple minor tweaks.  I can try setting up the automated build for you once it is merged.

File: Dockerfile
Comment (pataquets) [inline Dockerfile]: I prefer to use non-specific, somewhat common dir names instead. This is the dir name I've seen commonplace, for example in the Docker Python official image ```onbuild``` tags. Not sure if it's a best practice somewhere or just coincidence, since I'm not a very experienced "pythonist" :)
Also, I think that it is an obvious first try if you're jumping into a running container and trying luck when looking for the app dir.

**Review by gorzell — APPROVED** [review]
I am willing to leave it to personal preference.

**Comment by pataquets** [comment]
Willing to create an Automated Build on Docker Hub to provide an 'official' image? Thus, the image gets build on each commit and it serves as a hands-free distribution mechanism. See my OP above. Ping me if you need help.

**Comment by gorzell** [comment]
Thanks for the reminder, done.  https://hub.docker.com/r/keen/redis-statsd

**Comment by pataquets** [comment]
Awesome!

**Pull request merged**
